docs/capabilities.md

```diff
@@ -4,33 +4,103 @@ time.  Whether you are working with large legacy codebases, your own
 Atomist provides you with the tools to speed development, smooth
 operations, and close the code/runtime feedback loop.
 
-Atomist helps you develop better software more quickly in several
-important ways:
-
--   Linking related events from source code repositories, continuous
-    integration, testing, and runtime environments, presenting them in
-    a unified manner to appropriate team members to greatly reduce
-    context-switching costs and incident time-to-resolution.
--   Managing a service life cycle, from commits to CI builds to
-    deployments to alerts to corrective action
--   Automating common tasks in software development and operation,
-    for example:
-    -   Using Rug editors to modify code in a consistent mannger
-        across many repositories
-    -   Ensuring a project conforms with your standards using a Rug
-        reviewer
-
-# What you can do with Atomist
-## Rug is the eventing runtime that ties everything together
+## Bringing everything together
+
+Atomist consumes events from the platforms and tools you use every day.
+By linking related events from source code repositories, continuous
+integration, testing, and runtime environments, Atomist can present
+them in a unified manner to appropriate team members via direct or
+group messages.  The result is massive reductions in context-switching
+costs and and shorter incident time-to-resolution.
+
+<!-- *That rug really tied the room together, did it not?* -->
+
+Rug is the eventing runtime that ties everything together.  Rug not
+only understands individual events, but can learn relationships
+between events and respond to events with appropriate, automated
+actions.
+
 ## Development lifecycle automation
-### Event-driven actions: handlers
-### ChatDev and ChatOps: commands
-### Project updates: editors
-### Project compliance: reviewers
-### Project creation: generators
-## Events and code selection: tree expressions
-## Interfaces
-### XX Button XX
-### Bot
-### CLI
+
+Atomist helps you develop better software more quickly by greatly
+enhancing your ability to automate your software development
+lifecycle.  By intelligently linking your source code repository, your
+CI system, your deployment platform, your logs, and your alerting
+system, Atomist connects code changes to runtime changes and provides
+that information back, with its full context, back to the appropriate
+team member.  It can can even automate remediation, obviating the need
+to alert anyone.
+
+The Rug programming model provides several different types of Rugs to
+help you manage your development and operations lifecycle.
```

Comment:
perhaps something like: "The Rug language was created to enable building Rug scripts to help you streamline and manage your software delivery lifecycle." trying to  emphasize the act of writing Rug to solve these problems.